src/wmfcodec.c

```diff
@@ -59,6 +59,7 @@ gdip_getcodecinfo_wmf ()
 	return &wmf_codec;
 }
 
+#if DEBUG_WMF
```

Comment:
could you please move `#define GetParam(x,y) ...` from below into this ifdef as well since it's only used in DEBUG_WMF codepaths?

Reply:
Sure, fixed